Continuous integration.md

```diff
@@ -106,13 +106,13 @@ subprojects {
 }
 ```
 
-This will significantly speed up compilation of your apps on CircleCI.
+This will significantly speed up the compilation of your apps on CircleCI.
 
 ## Protected branches
 
-In the git flow model that is applied in Infinum, `dev` and `master` branches have a special role in the development and release cycle. It is expected that the code is not pushed directly to those two branches, therefore the branches need to be set as protected. Also, in order to make sure that the code in those two branches builds successfully at any time and always satisfies statical analysis rules and any other rules defined in `circle.yml`, Circle CI needs to be enabled on `dev` and `master`.
+In the git flow model that is applied in Infinum, the `dev` and `master` branche have a special role in the development and release cycle. It is expected that the code is not pushed directly to those two branches; therefore the branches need to be set as protected. Also, in order to make sure that the code in those two branches builds successfully at any time and always satisfies static analysis rules and any other rules defined in `circle.yml`, CircleCI needs to be enabled on `dev` and `master`.
```

Comment:
I don't understand why here the plural of the branch should be *branche*. Is there some explanation when and why the *s* is omitted or this is just a typo :) In this paragraph, we have 3 more occurrences of branches.

Reply:
It shouldn't be, it's a typo, I must have deleted the s by accident

Reply:
Cool thnx. I thought this was some exception in English language :)